Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2221,6 +2393,17 @@ bool CSSPropertyParser::consumeListStyleShorthand(bool important)
 
 bool CSSPropertyParser::parseShorthand(CSSPropertyID property, bool important)
 {
+    auto parseBorder = [&](CSSPropertyID widthProperty, CSSPropertyID styleProperty, CSSPropertyID colorProperty) {
```

Comment:
I’m not sure that matters. I don’t think the local variable for a lambda not called incurs overhead. If I made this a separate function it would need more arguments, but I guess I’ll do that, though.